ethereum/contracts/pyth/PythGovernanceInstructions.sol

```diff
@@ -43,9 +44,15 @@ contract PythGovernanceInstructions {
         address newImplementation;
     }
 
-    struct SetGovernanceDataSourcePayload {
-        PythInternalStructs.DataSource newGovernanceDataSource;
-        uint64 initialSequence;
+    struct TransferGovernanceDataSourcePayload {
+        // claimVaa is a VAA with TransferGovernanceDataSourceClaimOwnership
+        // governance instruction (header + payload) that is created from
+        // the second multisig
```

Comment:
"second multisig" -> "second governance source".

Might be worth adding a comment here about why we are doing this (i.e. to ensure that we can generate valid governance vaas from the new data source).

Reply:
Also would be worth a proper docstring here describing the outline and the purpose of action 1 and 2.